src/dedupe.rs

```diff
@@ -592,6 +620,13 @@ impl PartitionedFileGroup {
                     target: retained_file.clone(),
                     link: dropped_file,
                 }),
+                DedupeOp::Reflink if devices_differ => {
+                    panic!("reflink can not work across devices");
```

Comment:
Please don't use panic here. Just display a warning and continue with other files. Some does might be across devices but the others might not be.

Reply:
Oh absolutely not, that should have been an `unimplmented!("what now?")`. Also: `Reflink` or `RefLink` (but it is not based on two words, however it would be more consistent with `SofLink` and `HardLink`)?

Reply:
I prefer `RefLink`